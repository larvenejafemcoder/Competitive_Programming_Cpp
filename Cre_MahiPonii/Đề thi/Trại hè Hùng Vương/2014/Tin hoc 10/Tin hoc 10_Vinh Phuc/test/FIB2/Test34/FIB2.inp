100
10 38
1 1
40 52988628
8 1
1 1
33 2185228
6 13
0 1
0 1
21 1420
26 169439
27 21119
24 9673
7 12
18 583
35 220155
22 9631
42 400245480
35 11980183
44 263933324
8 17
28 343894
9 54
32 3387368
0 1
24 33846
15 568
0 1
3 1
28 290990
45 1038911625
13 56
0 1
19 3069
3 3
35 9234347
9 53
12 224
22 6760
15 891
13 200
14 529
16 307
20 3531
27 238362
10 30
44 948105147
37 37514790
33 1536695
40 5037629
35 8218341
42 407137490
24 11069
37 2107766
16 922
1 1
36 10664093
2 2
17 786
44 415276916
9 44
16 316
3 1
25 97145
26 88961
30 862981
41 109969616
44 298494258
31 342958
22 13679
45 1089883740
30 923280
36 15441344
28 260199
31 565092
7 18
12 173
39 82045115
34 5016894
45 466244660
39 14547884
12 13
38 17348677
44 1005436029
11 96
13 291
4 2
8 12
24 48604
0 1
45 1051658492
4 5
1 1
11 33
40 50996802
43 632230753
4 1
4 2
14 563
9 24
