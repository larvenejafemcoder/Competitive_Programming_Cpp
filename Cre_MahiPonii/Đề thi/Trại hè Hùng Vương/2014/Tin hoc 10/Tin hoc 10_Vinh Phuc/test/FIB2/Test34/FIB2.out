15
0
20239855
1
0
834683
5
1
1
543
64720
8067
3695
5
223
84092
3679
152880170
4576023
100813559
7
131356
21
1293860
1
12928
217
1
0
111148
396828930
22
1
1172
1
3527207
20
86
2582
340
77
202
117
1349
91046
12
362143941
14329375
586965
1924203
3139127
155512683
4228
805095
352
0
4073321
1
300
158621667
17
121
0
37106
33980
329630
42004656
114014661
130998
5225
416298545
352662
5898069
99387
215846
7
66
31338445
1916283
178089613
5556797
5
6626605
384042390
37
111
1
5
18565
1
401697800
2
0
13
19479045
241490659
1
1
215
9
